src/DIPS.Xamarin.UI/Controls/DatePicker/DatePicker.xaml

```diff
@@ -19,7 +19,6 @@
                     WidthRequest="{Binding Source={x:Reference DateLabel}, Path=Width}" />
         <Label x:Name="DateLabel"
                TextColor="{Binding Source={x:Reference codeBehind}, Path=LabelColor}"

```

Comment:
`Binding` had to be removed because it for some reason ignores the `TypeConverter` value if we use it. So the `FontSize` gets set in the code behind for each picker.